Doc/library/logging.handlers.rst

```diff
@@ -59,13 +59,25 @@ and :meth:`flush` methods).
       :meth:`close` method is inherited from :class:`~logging.Handler` and so
       does no output, so an explicit :meth:`flush` call may be needed at times.
 
+   .. method:: setStream(stream)
+
+      Sets the instance's stream to the specified value, if it is different.
+
+      :param stream: The stream that the handler should use.
```

Comment:
Do we support the `:param:` markup? I don't think I've seen it before in our docs.